doc/tutorials/enable_ivshmem.rst

```diff
@@ -0,0 +1,151 @@
+.. _enable_ivshmem:
+
+Enable Inter-VM Communication Based on Ivshmem
+##############################################
+
+Users can use inter-vm communication based on ``ivshmem`` dm-land solution or
+hv-land solution according to the usage scenario and both solutions can be
+used at the same time. The ivshmem device using dm-land cannot communicate
+with the ivshmem device using hv-land. For more details about ivshmem inter-vm
+communication, please refer to :ref:`Ivshmem Inter-VM Communication High Level
+Design <ivshmem-hld>`
+
+Ivshmem Dm-land Usage
+*********************
+add this line as an ``acrn-dm`` boot parameter::
+
+     -s slot,ivshmem,shm_name,shm_size
+
+where
+
+-  ``-s slot``  - Specify the virtual PCI slot number
+
+-  ``ivshmem``  - Virtual PCI device name
+
+-  ``shm_name`` - Specify a shared memory name. Post-launched VMs with the same
+   ``shm_name`` share a shared memory region. The ``shm_name`` needs to start
+   with ``sos:/`` prefix. For example, ``sos:/test``
+
+-  ``shm_size`` - Specify a shared memory size. The two communicating VMs must
+   define the same size.
+
+.. note:: This device can be used with Real-Time VM (RTVM) as well.
+
+Ivshmem Hv-land Usage
+*********************
+The ``ivshmem`` hv-land is not enabled by default, users need to enable
+``ivshmem`` hv-land through ACRN XML configuration(:ref:`ACRN
+Configuration Tool <acrn_configuration_tool>`) before using it.
+
+- Enable ``ivshmem`` hv-land in ACRN XML configuration file. For example, the
+  hybrid_rt scenario and whl-ipc-i5 board XML configuration file is located in
+  ``$PWD/misc/vm_configs/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml``
```

Comment:
``$PWD`` is where?  Let's just write this path as ``acrn-hypervisor/misc/...``

Reply:
Yes, it is better.